docs/queries/byrole.mdx

```diff
@@ -462,3 +462,20 @@ You can query a specific element like this
 ```js
 getByRole('alertdialog', {description: 'Your session is about to expire'})
 ```
+
+## Performance
```

Comment:
This looks good to me, but I'd like to keep it open for second reviewer.
One additional comment we might encourage is to use happy-dom as that should be faster.

Reply:
Sounds good, I'd be happy to add a note about happy-dom as I agree it is likely the best option if it works for people (I've seen some users report it isn't a viable option for them _yet_).

What do we think about adding something like this?

"Firstly, if you are using jsdom we'd encourage trying happy-dom as it is much more efficient. If this works for you it is your best option, improving performance while keeping test confidence.

Otherwise the suggestions below may help."

Reply:
Let's leave it like this for now, as this will likely break something.
We can always revisit it later.

Reply:
From my research, `happy-dom` is faster but lacks the fetureset of `JSDOM`, and that is reason folks often revert back to `JSDOM` after attempting a migration.